java/java-impl/src/com/intellij/codeInsight/javadoc/JavaDocExternalFilter.java

```diff
@@ -61,7 +61,7 @@ else if (href.startsWith("//")) {
             String[] parts = href.substring(2).split("/", 2);
             if (parts.length != 2) return href;
             Url relativeUrl = Urls.newUrl(scheme, parts[0], parts[1]);
```

Comment:
Thanks for the patch!

I am writing a test for this method and I was wondering about this logic. `Urls.newUrl("scheme", "autority", "path")` gives `scheme://autoritypath`, is that really what we want here?

`<a href="//example.com/about">` -> `<a href="https://example.comabout">`

Probably we should write `Urls.newUrl(scheme, parts[0], '/' + parts[1]);`?

Reply:
Sorry somehow I lost track of this. Thank you for following up on it :)